.github/workflows/topic-browser.yml

```
@@ -0,0 +1,48 @@
+    timeout-minutes: 30
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Go
+        uses: ./.github/actions/setup-go
+      - name: Install Ginkgo
+        run: go install github.com/onsi/ginkgo/v2/ginkgo@v2.22.1
+      - name: Test
+        env:
+          TEST_TOPIC_BROWSER: 1
+        run: make test-topic-browser 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)